tests/run-pass/time.rs

```diff
@@ -8,6 +8,14 @@ fn duration_sanity(diff: Duration) {
     assert!(diff.as_millis() < 500);
 }
 
+#[cfg(unix)]
```

Comment:
Please add a comment explaining why this runs only on Unix.

Reply:
I don't understand your comment. I expected it to say something like "sleeping not implemented on Windows", instead of talks about `c::Sleep`, whatever that is. Or is that what sleeps on Windows? Don't expect the reader to know such things.^^